avocado/core/nrunner.py

```diff
@@ -312,7 +313,7 @@ def pick_runner_class(self, runners_registry=None):
         raise ValueError('Unsupported kind of runnable: %s' % self.kind)
 
 
-class BaseRunner:
+class BaseRunner(metaclass=abc.ABCMeta):
```

Comment:
@tiagohonorato I wonder if this could simply be
```python
class BaseRunner(abc.ABC):
```
Direct (if not any) usage of metaclasses is discouraged and in this case not necessary since inheriting from `ABC` could rid is of such python internals and keep normal class inheritance on our side.

Reply:
Hi @pevogam, sorry for the wait here and thanks for pointing that out!

You are right. After researching a bit, since python 3.4, inheriting from ``ABC`` defines the metaclass implicitly:

```python

class ABC(metaclass=ABCMeta):
    """Helper class that provides a standard way to create an ABC using
    inheritance.
    """
    __slots__ = ()
```

So, yeah, I agree that keeping normal inheritance would be way better!

Reply:
Currently, from what I could find, these are the only places ```ABCMeta``` is used:

```
avocado/core/plugin_interfaces.py:19:class Plugin(metaclass=abc.ABCMeta):
avocado/core/nrunner.py:316:class BaseRunner(metaclass=abc.ABCMeta):
avocado/utils/datadrainer.py:32:class BaseDrainer(metaclass=abc.ABCMeta):
```

Reply:
Hi @tiagohonorato, sounds good. Nice to see that you are willing to take this a step further and cover other instances of this improvement!